### DISCUSSION TITLE
HMIP-RF USB Stick on Home Assistant not recognised after switching from HomeMatic to RaspberryMatic Addon

### DISCUSSION THREAD

**Discussion by northnorway** [OpenCCU/OpenCCU#1717] [🇺🇸 Q&A] [answered]
I was very glad, that RaspberryMatic is now available for Hass.IO - thanks for all the efforts behind! Unfortunately I am experiencing the problem described in the headline: My HMIP-RF-USB Stick is obviously not recognised. RaspberryMatic tells me, that no suituable communication device was found and in the log file I see the following entry: 
`Identifying Homematic RF-Hardware: ...HmRF: none, HmIP: none, OK
Updating Homematic RF-Hardware: HMIP-RFUSB: =>4.4.16, ERROR ( != 4.4.16)`
I used the HomeMatic Addon before without having this issue. I use the Odroid N2+ as hardware, the HMIP-RF-USB device is the original one provided by EQ3 / ELV. (and for the sake of completeness: I uninstalled the HomeMatic addon before installing raspberrymatic)
Any ideas what I could try to get it working? In advance, many thanks!

**Comment by northnorway** [comment, marked as answer]
Shame on me :-( Just forgot to disable protected mode.  Now it´s working perfect!

**Comment by jens-maus** [comment]
The disabling of protected mode is only temporary necessary because of a bug in the current stable version of the addon. This will not be necessary with the next version on.
